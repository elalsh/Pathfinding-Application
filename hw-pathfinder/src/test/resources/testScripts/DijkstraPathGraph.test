# Find a path with Dijkstra's algorithm when
# src and dest node are the same.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D
AddNode simpleGraph E

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph B C 1.0
AddEdge simpleGraph C D 50.0
AddEdge simpleGraph D E 8.0

FindPath simpleGraph A E