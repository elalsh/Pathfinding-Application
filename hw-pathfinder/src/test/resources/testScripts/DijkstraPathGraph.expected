# Find a path with Dijkstra's algorithm when
# src and dest node are the same.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph
added node E to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 1.000 from B to C in simpleGraph
added edge 50.000 from C to D in simpleGraph
added edge 8.000 from D to E in simpleGraph

path from A to E:
A to B with weight 10.000
B to C with weight 1.000
C to D with weight 50.000
D to E with weight 8.000
total cost: 69.000